### DISCUSSION TITLE
How to changing the pdf size to 80mm and A4?

### DISCUSSION THREAD

**Discussion by KingdomLew** [pdfme/pdfme#224] [General]
Hi guys, i'm new pdfme and I will need to generate PDF in 80mm size. By default the PDF will be in A4 size, how do I code to changing the PDF size to 80mm?

**Comment by hand-dot** [comment]
Please create a PDF with a size of 80mm in an application like Illustrator and load it as basePdf.

**Reply by KingdomLew** [reply]
Thank you, its help alot~
